devel/moth.py

```diff
@@ -106,6 +109,8 @@ def read_stream(self, stream):
                     self.summary = val
                 elif key == 'answer':
                     self.answers.append(val)
+                elif key == 'unlockedby':
+                    self.unlockers.append(val)
```

Comment:
Is it going to be a big PITA to pull the unlock-path work out from the dynamic grading? I feel like the unlock paths need a different lens, since they create interdependencies between puzzles when right now every puzzle is standalone

Reply:
No, it should be pretty quick to isolate the changes.  The logic is not interdependent between them.